src/pyipp/models.py

```diff
@@ -178,30 +178,50 @@ def merge_marker_data(data):
                 marker_highs.append(100)
                 marker_lows.append(0)
 
+        elif isinstance(data.get("marker-names"), str):
+            marker_names = [data["marker-names"]]
+
+            mlen = 1
+            marker_colors.append("")
+            marker_levels.append(-2)
+            marker_types.append("unknown")
+            marker_highs.append(100)
+            marker_lows.append(0)
+
         if isinstance(data.get("marker-colors"), list):
             for index, list_value in enumerate(data["marker-colors"]):
                 if index < mlen:
                     marker_colors[index] = list_value
+        elif isinstance(data.get("marker-colors"), str) and mlen == 1:
+            marker_colors[0] = data.get("marker-colors")
 
         if isinstance(data.get("marker-levels"), list):
             for index, list_value in enumerate(data["marker-levels"]):
                 if index < mlen:
                     marker_levels[index] = list_value
+        elif isinstance(data.get("marker-levels"), int) and mlen == 1:
+            marker_levels[0] = data.get("marker-levels")
 
         if isinstance(data.get("marker-high-levels"), list):
             for index, list_value in enumerate(data["marker-high-levels"]):
                 if index < mlen:
                     marker_highs[index] = list_value
+        elif isinstance(data.get("marker-high-levels"), int) and mlen == 1:
+            marker_highs[0] = data.get("marker-high-levels")
 
         if isinstance(data.get("marker-low-levels"), list):
             for index, list_value in enumerate(data["marker-low-levels"]):
                 if index < mlen:
                     marker_lows[index] = list_value
+        elif isinstance(data.get("marker--low-levels"), int) and mlen == 1:
```

Comment:
```suggestion
        elif isinstance(data.get("marker-low-levels"), int) and mlen == 1:
```

I think this dash is extra. Do you happen to have a debug dump from your printer? so I can put a proper test in place and make that's only issue.

Reply:
I may be able to use the log you provided to male a test. Looking into it now.

Reply:
Thanks for that. Herewith the debug dump from my printer: 
[printer-attributes.zip](https://github.com/ctalkington/python-ipp/files/11562121/printer-attributes.zip)